cmd/eulabia-sensor/main.go

```diff
@@ -38,9 +41,11 @@ func main() {
 	if err != nil {
 		log.Panicf("Failed to connect: %s", err)
 	}
-	c.Publish(topic, messages.Modify{
-		Message: messages.NewMessage("modify.sensor", "", ""),
-		ID:      *sensorID,
+	c.Publish("eulabeia/cmd/sensor/director", cmds.Modify{
+		Identifier: messages.Identifier{
+			Message: messages.NewMessage("modify.sensor", "", ""),
+			ID:      *sensorID,
+		},
```

Comment:
The topics are not set as said and documented. It should be:
<group>/<aggregate>/<event>/<destination>
Maybe we need some new events like modify and request. So this could be:
eulabeia/sensor/modify/<sensorID>

Reply:
It is in this case sensor is an aggregate